subprojects/logging/src/main/java/org/gradle/internal/logging/console/BuildStatusRenderer.java

```diff
@@ -16,32 +16,52 @@
 
 package org.gradle.internal.logging.console;
 
-import org.gradle.internal.logging.events.BatchOutputEventListener;
+import org.gradle.internal.logging.events.EndOutputEvent;
 import org.gradle.internal.logging.events.OperationIdentifier;
 import org.gradle.internal.logging.events.OutputEvent;
 import org.gradle.internal.logging.events.ProgressCompleteEvent;
 import org.gradle.internal.logging.events.ProgressEvent;
 import org.gradle.internal.logging.events.ProgressStartEvent;
+import org.gradle.internal.logging.events.BatchOutputEventListener;
 import org.gradle.internal.logging.text.Span;
 import org.gradle.internal.logging.text.Style;
 import org.gradle.internal.nativeintegration.console.ConsoleMetaData;
+import org.gradle.internal.time.TimeProvider;
 
 import java.util.Arrays;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
 
 public class BuildStatusRenderer extends BatchOutputEventListener {
     public static final String BUILD_PROGRESS_CATEGORY = "org.gradle.internal.progress.BuildProgressLogger";
+    private static final long RENDER_NOW_PERIOD_MILLISECONDS = 86;
```

Comment:
I use a similar value to the default for throttling. When I wrote is my memory failed me and put `86` instead of `85`. I changed it back to `85`. I prefer such value as it won't aligned on a round number, so the displayed time won't look odd. We can always change the display and use a more round number.